apps/site/lib/site_web/templates/cms/paragraph/_fare_card.html.eex

```diff
@@ -25,6 +28,7 @@
 
   media = case fare do
     %Fares.Summary{duration: :weekend} -> ["CharlieTicket, Cash, or ", mticket_html]
+    %Fares.Summary{modes: [:ferry]} -> [mticket_html, " or Paper Ferry Ticket"]
```

Comment:
I know it's not done on line 30, but would it be more dynamic to use `media/1` here instead of hard-coding the string? It would reduce the number of places it's hard coded.